usr/src/cmd/pkgsrc-wrapper/pkgsrc-wrapper.c

```diff
@@ -0,0 +1,181 @@
+/*
+ * This file and its contents are supplied under the terms of the
+ * Common Development and Distribution License ("CDDL"), version 1.0.
+ * You may only use this file in accordance with the terms of version
+ * 1.0 of the CDDL.
+ *
+ * A full copy of the text of the CDDL should have accompanied this
+ * source.  A copy of the CDDL is also available via the Internet at
+ * http://www.illumos.org/license/CDDL.
+ */
+
+/*
+ * Copyright 2023 MNX Cloud, Inc.
+ */
+
+/*
+ * Use a C program instead of a shell script to exec() something in
```

Comment:
This probably shouldn't mention the shell script, since it's not something that ever shipped and won't even exist in the master branch history.

Reply:
Will fix.